project-base/k8s/services/webserver.yml

```diff
@@ -0,0 +1,12 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name:  webserver
+spec:
+  selector:
+    app: webserver
+  type: NodePort
+  ports:
+  - name: http
+    port:  8080
```

Comment:
Why use 8080? I would be rather for the standard 80, or 8000 (the same as the default if installed locally)